README.md

```diff
@@ -76,14 +76,21 @@ The basic workings of this filter are done via the [`filters::FilterBase` API](h
     once. If this is true, the processing pipeline expects the pointcloud
     to have fields int32 index, float32 stamps, and float32 vp_x, vp_y
     and vp_z viewpoint positions. If one of these fields is missing,
-    computeMask() throws runtime exception.
+    computeMask() throws runtime exception. For the LaserScan version, this
```

Comment:
This comment looks misplaced - it was probably meant to be appended to filtering frame docs (and I wrongly suggested you to add it to fixed frame docs, but it's not even there correctly). Also, it's missing the point that you can only do so for scans captured whole in the same time instant.

Reply:
Could you update the description (to prevent a ping-pong conversation)?